content/blogposts/2022/cloud-managed-instances-for-enterprise.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Secure and scalable Sourcegraph Cloud managed instances for enterprise"
+publishDate: 2022-09-27T10:00-07:00
+description: Sourcegraph Cloud, the single-tenant, dedicated cloud version of Sourcegraph, is now generally available.
+tags: [blog]
+slug: "enterprise-cloud"
+published: false
+heroImage: 
+socialImage: 
+changelogItems:
+---
+
+Soucegraph has long existed as self-hosted software, letting our customers confidently and securely run Sourcegraph in their own environments. After almost a decade of developing self-hosted Sourcegraph, we’ve gained the trust of some of the most sophisticated companies and development teams in the world, allowing us to work with their most sensitive IP: their code.
+
+At the same time, we’ve seen immense demand for Cloud-based Sourcegraph in recent years. Developers want a solution that can be spun up quickly, upgraded regularly, and maintained effortlessly without compromising security.
+
+In 2020, we experimented with deploying Cloud-based Sourcegraph to fit these needs, which we called “managed instances.” We ran these instances as single-tenant, isolated deployments in GCP, and the demand was so strong that almost 10% of our customers are using this solution today. Since then, we’ve invested significantly in this single-tenant architecture solution, making it more scalable and bringing it to full feature parity with Sourcegraph Self-hosted. 
```

Comment:
```suggestion
In 2020, we experimented with deploying cloud-based Sourcegraph to fit these needs, which we called “managed instances.” We ran these instances as single-tenant, isolated deployments in GCP, and the demand was so strong that almost 10% of our customers are using this solution today. Since then, we’ve invested significantly in this single-tenant architecture solution, making it more scalable and bringing it to full feature parity with Sourcegraph Self-hosted. 
```